### PR TITLE
SBERDOMA-522 documentation draft of migrations guide

### PR DESCRIPTION
**Pull request by AntonAL**
Working with migrations has much of pitfalls, that spread by word across team.
Let's keep all of them in one place.
Outlined points are need to be discussed.

### CODE REVIEW

File: apps/condo/docs/migrations.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+Migrations guide
+=====
+
+Migrations should not be changed between releases ;)
+
+## Changed a schema during work in progress — recreate corresponding migration
+
+Suppose, You changed structure of a schema, you are working with.
+You need the changes to be reflected in a corresponding migration file.
+Don't change migration file by hands.
```
Comment (pahaz) [inline apps/condo/docs/migrations.md:10]:
```suggestion
Don't change migration files by hands If you don't really understand how the kmigrator works.
```

File: apps/condo/docs/migrations.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
+Migrations guide
+=====
+
+Migrations should not be changed between releases ;)
```
Comment (pahaz) [inline apps/condo/docs/migrations.md:4]:
```suggestion
Migrations should not be changed if they have already been merged in master ;)
```

File: apps/condo/docs/migrations.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+```
+docker-compose run app yarn workspace @app/condo makemigrations
+```
+
+## Errors and solutions
+
+### CommandError: Conflicting migrations detected
+
+If you faced this message, do:
```
Comment (pahaz) [inline apps/condo/docs/migrations.md:74]:
```suggestion
It's better to roll back your migration and recreate it. But sometimes you should do:
```

**Review by pahaz — APPROVED** [review]

**Review by dkoviazin — APPROVED** [review]
Ok. But better to drop database, run migrate, then makemigrations. And do it every time you are rebasing to master 😆

**Pull request merged**